[2023-10-20T00:00:01.818+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple.pokelist scheduled__2023-10-19T00:00:00+00:00 [queued]>
[2023-10-20T00:00:01.827+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple.pokelist scheduled__2023-10-19T00:00:00+00:00 [queued]>
[2023-10-20T00:00:01.827+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-10-20T00:00:01.842+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): pokelist> on 2023-10-19 00:00:00+00:00
[2023-10-20T00:00:01.850+0000] {standard_task_runner.py:57} INFO - Started process 5053 to run task
[2023-10-20T00:00:01.860+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple', 'pokelist', 'scheduled__2023-10-19T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/pokelist.py', '--cfg-path', '/tmp/tmpdgcwmozz']
[2023-10-20T00:00:01.867+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask pokelist
[2023-10-20T00:00:01.939+0000] {task_command.py:416} INFO - Running <TaskInstance: simple.pokelist scheduled__2023-10-19T00:00:00+00:00 [running]> on host f04a086795a0
[2023-10-20T00:00:02.041+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple' AIRFLOW_CTX_TASK_ID='pokelist' AIRFLOW_CTX_EXECUTION_DATE='2023-10-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-19T00:00:00+00:00'
[2023-10-20T00:00:02.044+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-20T00:00:02.045+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/scripts/pokelist.py']
[2023-10-20T00:00:02.067+0000] {subprocess.py:86} INFO - Output:
[2023-10-20T00:00:02.185+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-10-20T00:00:02.186+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/pokelist.py", line 2, in <module>
[2023-10-20T00:00:02.187+0000] {subprocess.py:93} INFO -     from pyspark.sql import SparkSession
[2023-10-20T00:00:02.187+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'pyspark'
[2023-10-20T00:00:02.189+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-10-20T00:00:02.213+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-10-20T00:00:02.219+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=simple, task_id=pokelist, execution_date=20231019T000000, start_date=20231020T000001, end_date=20231020T000002
[2023-10-20T00:00:02.235+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 41 for task pokelist (Bash command failed. The command returned a non-zero exit code 1.; 5053)
[2023-10-20T00:00:02.268+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-20T00:00:02.298+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
